DEVELOPER_GUIDE.md

```diff
@@ -38,7 +46,9 @@ This document will walk you through on what's needed to start contributing code
 
 ### Setup
 
-To develop OSB properly, it is recommended that you fork the official OpenSearch Benchmark repository.
+To develop OSB properly, it is recommended that you fork the official OpenSearch Benchmark repository. 
+
+For those working on WSL2, it is recommended to clone the repository and set up the working environment within the Linux subsystem. Refer to the guide for setting up WSL2 on [Visual Studio Code](https://code.visualstudio.com/docs/remote/wsl) or [PyCharm](https://www.jetbrains.com/help/pycharm/using-wsl-as-a-remote-interpreter.html#create-wsl-interpreter). 
```

Comment:
It would be good to indicate that WSL is a supported platform for development, if that is the case.  Some documentation on whether it is WSL1 or WSL2 (or both) would be helpful.

Reply:
As far as I have tested, it is possible to work on both. Shall I just replace WSL2 with WSL in the documentation, or explicitly say that both are ok to work with.

Reply:
I recommend explicitly saying that both WSL and WSL2 work with OSB